shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1,37 +1,94 @@
 use crate::helpers::ShotoverManager;
-use cassandra_cpp::{stmt, Session};
 use serial_test::serial;
 use test_helpers::docker_compose::DockerCompose;
 
-use crate::cassandra_int_tests::{assert_query_result, cassandra_connection, ResultValue};
+use crate::cassandra_int_tests::cassandra_connection;
 
-fn test_create_keyspace(session: Session) {
-    assert_query_result(
-        &session,
+mod keyspace {
+    use cassandra_cpp::{stmt, Session};
+
+    use crate::cassandra_int_tests::{assert_query_result, ResultValue};
+
+    fn test_create_keyspace(session: &Session) {
+        assert_query_result(
+        session,
         stmt!(
             "CREATE KEYSPACE IF NOT EXISTS cycling WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };"
```

Comment:
is it possible to first delete the keyspace if it already exists?
Otherwise we might get false positives or just strange failures if a previous test case creates a keyspace with the same name.
Its still early days for these tests so happy to leave it for later if its not a simple fix.

Reply:
I'll change the keyspace name `keyspace_tests` and similar for the other tests to prevent this. A bit of discipline with the test names we use should avoid any issues like that.

Reply:
Can we remove the `IF NOT EXISTS` so it blows up if it does happen to exist?